samples/apps/chat-summary-webapp-react/src/hooks/SemanticKernel.ts

```diff
@@ -50,16 +50,25 @@ export class SemanticKernel {
             headers.append(SK_HTTP_HEADER_MSGRAPH, keyConfig.graphToken);
         }
 
-        const response = await fetch(`${this.serviceUrl}${commandPath}`, {
-            method: method ?? 'GET',
-            headers,
-            body: JSON.stringify(body),
-        });
+        try {
+            const response = await fetch(`${this.serviceUrl}${commandPath}`, {
+                method: method ?? 'GET',
+                headers,
+                body: JSON.stringify(body),
+            });
 
-        if (!response.ok) {
-            throw response.statusText;
-        }
+            if (!response.ok) {
+                throw response.statusText + " => " + await response.text();
+            }
 
-        return (await response.json()) as T;
+            return (await response.json()) as T;
+        } catch (e) {
+            var additional_error_msg = ''
+            if (e instanceof TypeError) {
+                // fetch() will reject with a TypeError when a network error is encountered.
+                additional_error_msg = '\n\nPlease check you have the function running and that it is accessible by the web app'
```

Comment:
nit: I would remove the word "web" considering that this function can power any sort of app

Reply:
Removed